doc/man3/SSL_send_alert.pod

```diff
@@ -0,0 +1,46 @@
+=pod
+
+=head1 NAME
+
+SSL_send_alert - send ssl alert to ssl connection
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_send_alert(SSL * ssl, int level, int desc);
+
+=head1 DESCRIPTION
+
+SSL_send_alert() sends alert to the ssl connection, and terminates session in case level is B<SSL3_AL_FATAL>.
+
+=head1 RETURN VALUES
+
+The following return values can occur:
+
+=over 4
+
+=item E<gt> 0
+
+The send alert operation was succeeded, the return value is the number of bytes actually written to the TLS/SSL connection. (see L<SSL_write(3)>)
```

Comment:
s/succeeded/successful/

Reply:
It does not seem very useful to know the number of bytes written. Perhaps we should just return 1 on success.